pkgs/development/tools/simavr/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, avrgcclibc, libelf, which, git, pkgconfig, freeglut
+, mesa }:
+
+stdenv.mkDerivation rec {
+  name = "simavr-${version}";
+  version = "1.3";
+  enableParallelBuilding = true;
+
+  libPath = stdenv.lib.makeLibraryPath [
+    libelf
+  ];
+
+  src = fetchFromGitHub {
+    owner = "buserror";
+    repo = "simavr";
+    rev = "51d5fa69f9bc3d62941827faec02f8fbc7e187ab";
+    sha256 = "0k8xwzw9i6xsmf98q43fxhphq0wvflvmzqma1n4jd1ym9wi48lfx";
+  };
+
+  buildFlags = "AVR_ROOT=${avrgcclibc}/avr SIMAVR_VERSION=${version}";
+  installFlags = buildFlags + " DESTDIR=$(out)";
+
+  postFixup = ''
+    patchelf --set-rpath "$libPath:$out/lib" "$out/bin/simavr"
```

Comment:
@joachifm thank you.  Now I don't blindly overwrite the rpath, but add `$out/lib` to it.  And got rid of explicit libelf dependency.